man/docker-info.1.md

```diff
@@ -51,7 +51,14 @@ Here is a sample output:
     Architecture: x86_64
     CPUs: 1
     Total Memory: 2 GiB
-
+	Name: docker
+    ID: I54V:OLXT:HVMM:TPKO:JPHQ:CQCD:JNLC:O3BZ:4ZVJ:43XJ:PFHZ:6N2S
```

Comment:
I think this also needs formatting. Can we make sure the whole example is properly aligned?